bob-apps/bin/bobd.rs

```diff
@@ -34,7 +34,16 @@ async fn main() {
     log4rs::init_file(node.log_config(), Default::default()).expect("can't find log config");
 
     let mut mapper = VirtualMapper::new(&node, &cluster).await;
-    let mut addr = node.bind().to_socket_addrs().unwrap().next().unwrap();
+
+    let mut addr = if let Some(addr) = node.bind_to_ip_address() {
+        addr
```

Comment:
Sanity check is required: if `bind_to_ip_address` is specified and `node` address is also specified as ipv4 and they are different, then log error and panic

Reply:
Done